amundsen_application/api/issue/issue.py

```diff
@@ -0,0 +1,78 @@
+from flask import current_app as app
+from flask import jsonify, make_response, Response
+from flask_restful import Resource, reqparse
+from http import HTTPStatus
+import logging
+
+from amundsen_application.issue_tracker_clients import get_issue_tracker_client
+from amundsen_application.issue_tracker_clients.issue_exceptions import IssueConfigurationException
+from jira import JIRAError
+
+LOGGER = logging.getLogger(__name__)
+
+
+class IssuesAPI(Resource):
+    def __init__(self) -> None:
+        self.reqparse = reqparse.RequestParser()
+        # declared as none initially so that it doesn't throw configuration errors until a call is made
+        self.client = None
+
+    def get(self) -> Response:
+        """
+        Given a table key, returns all tickets containing that key. Returns an empty array if none exist
+        :return: List of tickets
+        """
+        try:
+            self.client = get_issue_tracker_client()
+            if not app.config['ISSUE_TRACKER_CLIENT_ENABLED']:
+                message = 'Issuing tracking is not enabled. Request was accepted but no issue will be returned.'
+                logging.exception(message)
+                return make_response(jsonify({'msg': message}), HTTPStatus.ACCEPTED)
```

Comment:
Should we return `404 NOT_FOUND`?

Reply:
i was following the pattern from another method in the project where if it wasn't configured, it returned accepted. i think 404 would imply there just aren't any issues right? so i'm leaning accepted seems better

Reply:
Got it. Sounds good.